scripts/configurator/parser/cmake_row.py

```diff
@@ -45,8 +44,8 @@ def __init__(self, raw_row: str, row_number: int, comment: str = ""):
       """
       row  = raw_row.strip()
       matches: Optional[Match[str]]  = re.match(self.regex_expression, row)
-      self.row_type = row_type.NOT_PARSABLE
-      self.val: str | bool = ""
+      self.row_type: row_type | None = None
```

Comment:
Good but where do we fail? At the end of the `if/elif` sequence there must be an `else: raise RuntimeError("...")`. We cannot just silently leave the `row_type` as None.